doc/adr/0009-make-offline-mode-file.md

```diff
@@ -0,0 +1,49 @@
+# 9. make-offline-mode-file
+
+Date: 2021-06-14
+
+## Status
+
+Accepted
+
+## Context
+
+In LinShare Flutter application, we can make offline mode files one by one from "My Space", "Received Shares" or "Shared Spaces" with the library [`sqflite`](https://pub.dev/packages/sqflite):
+
+SQLite plugin for Flutter. Supports iOS, Android and MacOS.
+
+- Support transactions and batches
+- Automatic version management during open
+- Helpers for insert/query/update/delete queries
+- DB operation executed in a background thread on iOS and Android
+
+Usage 
```

Comment:
Please describe the scheme for the tables which you use for current implementation? Fields, DB, ...